internal/host/docker_unix.go

```diff
@@ -15,16 +15,13 @@ import (
 // Docker uses docker exec to target a running container.
 type Docker struct {
 	cmdHost
-	// Container name
-	Container string
-	// Username or UID (format: "<name|uid>[:<group|gid>]")
-	User string
+	// Username or UID (format: "<name|uid>[:<group|gid>]") and Container Name ( eg: root@ubuntu )
+	Connection string
```

Comment:
Nit: This name sounded odd to me at first... `docker exec` has `--user` (which oddly can be... a group as well...)) and `IMAGE`, and... there's not really a connection going on (necessarily). I don't have a better name though...

Reply:
I was thinking about `ConnectionString` such as used for Postgres, MySQL and some times Redis. That would make it bit more comprehensive. WDYT ?

Reply:
After writing some of the comments below, regarding parsing this, I wonder if keeping user & image here as before, and just moving this logic to the frontend (CLI) would be better. Naming stuff is hard, lol, doing so avoids this conversation altogether.
(don't forget, it is a nit comment)

Reply:
We need to remember, this is not the IMAGE name, it is the container name or ID. Better not mix taxonomies.
Now I am okay with `ConnectionString`, it makes much more sense anyway.

Reply:
I still don't have a better name, and... not worth the discussion: go for whatever you think is best, I trust you :-D